forge.config.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = {
+  packagerConfig: {
+    icon: './src/resources/icons/logo',
+    executableName: 'cucumber-forge',
+  },
+  makers: [
+    {
+      name: '@electron-forge/maker-squirrel',
+      config: {
+        title: 'Cucumber Forge',
+        name: 'cucumber_forge_desktop',
+        exe: 'cucumber-forge.exe',
+        setupIcon: './src/resources/icons/logo.ico',
+        loadingGif: './src/resources/loading.gif',
+        iconUrl: 'https://raw.githubusercontent.com/cerner/cucumber-forge-desktop/master/src/resources/icons/logo.ico',
+      },
+    },
+    {
+      name: '@electron-forge/maker-dmg',
+      config: {
+        name: 'Cucumber Forge',
+        icon: './src/resources/icons/logo.icns',
+      },
+    },
+    {
+      name: '@electron-forge/maker-zip',
+      platforms: [
+        'linux',
+      ],
+    },
+  ],
+  publishers: [
+    {
+      name: '@electron-forge/publisher-github',
+      config: {
+        repository: {
+          owner: 'cerner',
+          name: 'cucumber-forge-desktop',
+        },
+        draft: true,
+      },
+    },
+  ],
+  hooks: {
+    postMake: async (forgeConfig, makeResults) => {
+      // Rename the windows and OSX executables for consistency
+      await makeResults.filter(result => result.platform === 'win32' || result.platform === 'darwin')
+        .forEach((makeResult) => {

```

Comment:
is this just a middle process? if so, could you move the closure to an external const and then just drop the const in the forEach parameter?